pkg/operator/operator.go

```diff
@@ -210,6 +221,10 @@ func LoadOperatorConf(cmd *cobra.Command) *Conf {
 		c.Deployment.Spec.Template.Spec.ImagePullSecrets =
 			[]corev1.LocalObjectReference{{Name: options.ImagePullSecret}}
 	}
+
+	c.SecurityContextConstraints.Name = c.SA.Name
```

Comment:
why do we need to set the name dynamically and not in the yaml?

Reply:
right, I can remove both